src/lib/mongoBackend/mongoConnectionPool.cpp

```diff
@@ -191,9 +193,14 @@ static DBClientBase* mongoConnect
 
   if (connected == false)
   {
-    LM_E(("Database Error (connection failed, after %d retries: '%s')", retries, err.c_str()));
+    char cV[64];
```

Comment:
Not for every sentence, but we should find a size large enough so all the sentences fit. Or we could even have step limits:

SHORT_SENTENCE_MAX_LENGTH
LONG_SENTENCE_MAX_LENGTH
VERY_LONG_SENTENCE_MAX_LENGTH

The final purpose of this would be to have code more ordered and centralized, avoiding "random sizes" every time and a centralized place to implement bugfixing.

However, not so important. Keep 64 there is you prefer so.